compiler/rustc_borrowck/src/diagnostics/mutability_errors.rs

```diff
@@ -173,7 +188,10 @@ impl<'a, 'tcx> MirBorrowckCtxt<'a, 'tcx> {
             } => bug!("Unexpected immutable place."),
         }
 
-        debug!("report_mutability_error: item_msg={:?}, reason={:?}", item_msg, reason);
+        debug!(
+            "report_mutability_error: item_msg and reason is moved to a struct due to diagnosing migration: {:?}",
+            &diagnosing
+        );
```

Comment:
```suggestion
        debug!(?diagnosing);
```
nit: if it makes sense to keep this debug output